code/modules/mob/living/bot/trashbot.dm

```diff
@@ -0,0 +1,138 @@
+/mob/living/bot/trashbot
+	name = "Trashbot"
+	desc = "A robot that picks up trash off of the floor."
```

Comment:
A robot that picks trash up off of the floor.

more grammatically accurate i believe

Reply:
Noted